config/cmake/installer/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@
 # For details, see the LICENSE file.
 # LLNS Copyright End
 
+message(FATAL_ERROR "This file is significantly out-of-date with the root CMakeLists.txt")
```

Comment:
This file started out as almost the same as the root CMakeLists.txt; it's very different now. Is it worth keeping around?

Reply:
this whole folder is a copy of the original effort at making an installer for GridDyn.  it shouldn't be used but I was keeping a few things related to NSIS around since I may use that again some day.    Most can likely be discarded.  But I can take a look at that later.